tests/containers/integration-test-packit

```diff
@@ -0,0 +1,24 @@
+FROM quay.io/bluechi/integration-test-base:latest
```

Comment:
Couldn't we reuse the `integration-test-snapshot` image and pass in the `$copr_repo` also there? (which would be`@centos-automotive-sig/bluechi-snapshot`)

Reply:
For each PR packit creates new temporary COPR repo for unmerged RPMs. For this PR it's https://copr.fedorainfracloud.org/coprs/packit/eclipse-bluechi-bluechi-488/ that's why I need to pass different copr repo for each PR.

Only after merge RPMs will be pushed to `@centos-automotive-sig/bluechi-snapshot`

Reply:
Fixed